.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: ruby
-sudo: false
+# Workaround for https://github.com/travis-ci/travis-ci/issues/8836
```

Comment:
I have subscribed to the above GitHub issue so that I am notified of updates. Once this has been resolved, I will put in a follow-up PR to revert this change.